digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -388,4 +387,30 @@ public static void collectExceptionMessage(StringBuilder sb, Throwable ex, Strin
             collectExceptionMessage(sb, t, used);
         }
     }
+
+    private static final String[] CONFIG_KEYS_FOR_LOGGING = {
+            "project_id",
+            "session_id",
+            "session_time",
+            "attempt_id",
+            "task_name",
+            "last_session_time",
+            "last_executed_session_time",
+            "next_session_time",
+            "session_uuid",
+            "timezone"
+    };
+
+    @VisibleForTesting
+    public Config filterConfigForLogging(Config src)
+    {
+        Config dst = cf.create();
+        for (String key : CONFIG_KEYS_FOR_LOGGING) {
+            Optional<Object> v = src.getOptional(key, Object.class);
+            if (v.isPresent()) {
+                dst.set(key, v.get());
```

Comment:
Config.set() just shares the reference to the values without copying data and won't consume memory a lot. Looks good.